### DISCUSSION TITLE
[Regression] Setting temperature no longer works with GoControl GC-TBZ48 (possibly due to `setpointScale` being `undefined` for thermostat setPoints)

### DISCUSSION THREAD

**Discussion by alexschneider** [zwave-js/zwave-js#7028] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
Setting the temperature of the thermostat via Z-Wave UI no longer works after updating to the newest version. Downgrading to my previous version works as expected. 

What did you expect to happen instead?
Setting the temperature to actually set the temperature. 

Try setting the temperature using the set temperature command class. 

Anything else we should know?

### Software versions

Old:
zwave-js-ui: 9.14.6
zwave-js: 12.12.4
home id: 3806735750
home hex: 0xe2e62d86

New:
zwave-js-ui: 9.16.0
zwave-js: 13.0.0
home id: 3806735750
home hex: 0xe2e62d86

### Device information

GoControl GC-TBZ48
Node 065

I tried excluding/reincluding and that has no effect. Only downgrading works. 

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

Old version (working)

[zwavejs_old.log](https://github.com/user-attachments/files/16289780/zwavejs_old.log)

New version (not working)

[zwavejs_new.log](https://github.com/user-attachments/files/16289781/zwavejs_new.log)

**Comment by alexschneider** [comment]
Based on some discussion in discord, including an additional log for the downgraded version:

[zwavejs_current(8).log](https://github.com/user-attachments/files/16301118/zwavejs_current.8.log)

And a new log including an interview 

[zwavejs_current.log](https://github.com/user-attachments/files/16301124/zwavejs_current.log)

**Reply by alexschneider** [reply]
And here's another log from the old version including the reinterview

[zwavejs_current.log](https://github.com/user-attachments/files/16301189/zwavejs_current.log)

It seems that `setPointScale` is undefined in the new version but is defined in the old version.

**Comment by AlCalzone** [comment, marked as answer]
Fixed by #7031
